txp/Pos/Txp/Toil/Utxo/Functions.hs

```diff
@@ -186,7 +186,7 @@ verifyInputs VTxContext {..} resolvedInputs TxAux {..} = do
         -> (TxIn, TxOutAux) -- ^ Input and corresponding output data
         -> TxInWitness
         -> m ()
-    inputPredicates i (txIn@TxIn{..}, toa@(TxOutAux txOut@TxOut{..} _)) witness = do
+    inputPredicates i (txIn, toa@(TxOutAux txOut@TxOut{..} _)) witness = do
```

Comment:
I don't understand, where is the logic we were discussing?
If there are inputs with unknown types, our behaviour depends on `vtcVerifyAllIsKnown`:
1. If it's `True`, we must report an error that input type is unknown (it should be checked explicitly to have clear error messages).
2. If it's `False`, we should ignore most of the checks (maybe all checks). For instance, we should ignore sum check, etc.